connectivity/conn_http.c

```diff
@@ -16,11 +16,11 @@
 #include <sys/socket.h>
 #include <unistd.h>
 
-#include "ca_cert.h"
 #include "conn_http.h"
 
 #define MAX_PORT_LEN 5
 #define MAX_CONTENT_LENGTH_LEN 5
+#define CA_PATH "fullchain6.pem"
```

Comment:
Prepare utilities to convert arbitrary file into C header to avoid file operations.

Reply:
The certificate will be put into secure storage. Should we convert it to C header here.
Related at #19 .

Reply:
If something is about to be preserved in secure storage, don't treat it as mixture.